app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -92,17 +92,28 @@ def show
     param :id, :identifier, :desc => N_("content view numeric identifier"), :required => true
     param :name, String, :desc => N_("module name to restrict modules for"), :required => false
     def available_puppet_modules
```

Comment:
Given the extra logic added to available_puppet_modules and add_use_latest_records, would it be possible to add a couple of controller tests?